meta-psa/recipes-security/psa-trusted-storage-linux/psa-trusted-storage-linux_1.0.0.bb

```diff
@@ -8,21 +8,31 @@ HOMEPAGE = "https://github.com/ARMmbed/psa_trusted_storage_linux.git"
 DESCRIPTION = "A Linux C library reference implementation of the PSA protected_storage.h API"
 SECTION = "libs"
 
+inherit systemd
+
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = " \
     file://LICENSE;md5=302d50a6369f5f22efdb674db908167a \
     file://apache-2.0.txt;md5=3b83ef96387f14655fc854ddc3c6bd57 \
     "
 
-SRC_URI = "git://git@github.com/armmbed/psa_trusted_storage_linux.git;protocol=ssh;nobranch=1"
+SRC_URI = " \
+    git://git@github.com/armmbed/psa_trusted_storage_linux.git;protocol=ssh;nobranch=1 \
+    file://psa-ecryptfs.service \
+    file://psa-ecryptfs-init.sh \
+    "
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
I think this line is not needed since it is not a `bbappend` and bitbake already adds the `${THISDIR}/files` directory in the default search path.